### PR TITLE
WIP: Detailed documentation on creating a new seed cluster

### PR DESCRIPTION
**Pull request by scruplelesswizard**
This is currently a work-in-progress. Please add any comments regarding any steps that are not clear, or any wording/information that is confusing

### CODE REVIEW

File: config/README.md [hunk 1]
```diff
@@ -1,17 +1,202 @@
...
+    - `bare-metal`
+
+    **The value should be quoted.**
+
+    Example:
+    > `- URL: "http://heketi:8080"`
+
+- #### `NodePortExposer` _(optional)_
+
+  The NodePort Exposer component replaces K8Sniff in newer deployments. Additionally it uses the block-style syntax for it's keys and values.
```
Comment (mrIncompetent) [inline config/README.md:134]: It is only required in Cloud deployments where NodePorts are by default not reachable from the outside.

**Comment by mrIncompetent** [comment]
Execept for one nit, dop af

**Comment by scheeles** [comment]
Can we move all official doc into https://github.com/kubermatic/docs

File: config/README.md (hunk 1 above)
Comment (scruplelesswizard) [reply config/README.md:134]: Covered

**Comment by scruplelesswizard** [comment]
Yes, will move it after the merge (in order to maintain the discussion and review)

**Comment by scheeles** [comment]
LGTM, but can you make the DNS part generic and not mention OVH

**Comment by scruplelesswizard** [comment]
Yes, this is just a WIP so far. I will get to the DNS stuff after datacenters and adding the installer is done ;)

File: config/README.md [hunk 2]
```diff
@@ -1,17 +1,217 @@
...
+  This is the URL to access the dashboard of the cluster. This will also be the subdomain which the client clusters will be assigned under (_i.e. <cluster-address.subdomain.domain.tld>_)
+
+  **The value should be quoted.**
+
+  Example:
+  > `KubermaticURL: "cloud.kubermatic.io"`
+
+- #### `Kubeconfig` _(required)_
+
+  This is a standard [Kubeconfig][2]. The correct context for the cluster does not need to be set, but the context name must match the key for the seed cluster definition in the `KubermaticDatacenters`
```
Comment (metalmatze) [inline config/README.md]:
```diff 
- in the `KubermaticDatacenters`
+ in the `KubermaticDatacenters` parameter
```

File: config/README.md [hunk 3]
```diff
@@ -1,17 +1,217 @@
...
+
+  When defining multiple seed clusters you must have a valid context for each seed cluster in the `Kubeconfig` with the context names must match to the correct keys in `KubermaticDatacenters`
+
+  **The value should be quoted.**
+
+  > The `Kubeconfig` value must be base64 encoded, without any linebreaks
```
Comment (metalmatze) [inline config/README.md]: Maybe add an example on how to do that with the CLI? Something like: `base64 ~/.kube/config`

**Review by metalmatze — DISMISSED** [review]

**Review by mrIncompetent — DISMISSED** [review]

**Review by mrIncompetent — APPROVED** [review]

**Pull request merged**